content/docs/guide/map_submission/map_zoning.md

```diff
@@ -9,60 +9,420 @@ weight: 4
 
-You can set map-wide properties by clicking on any of the tracks in the list:
+{{<hint info>}}
 
-![Map Properties](/images/map_porting/editor_track.png)
+In the zoning menu **stages** are called **segments**.  
+That is because of difference in uses between gamemodes.
 
-When you are done zoning your map, you can save your changes by clicking the blue "Save Zones" button. Zones will be saved into your `maps/zones/local` folder.
+{{</hint>}}
 
-For more advanced zoning options, please see our [Advanced In-Game Zoning Guide](). **TODO: Add link**
+### Right section
+This section is for creating **checkpoints**.  
+**Checkpoints** measure time for **comparisons** but **don't have leaderboards**.
 
-# Hammer Zoning
+### Bottom section
+This section lists additional properties of zones.  
+They are covered [later in the guide](#setting-zone-properties).
 
-If you would rather zone maps in Hammer, follow this guide instead.
+![Zoning Menu Showcase](/images/map_zoning/zoning_menu_showcase.png)
+
+# Basic Zoning
+
+## Creating the Start Zone
+Every **stage** needs a start zone. 
+1. Noclip to the start of the map
+    - **g** by default or `noclip` in console 
+1. Click **+ Main**
+    - This will automatically create the **main track**, **stage 1** and put you in the **zoning mode** 
+2. Select one corner of the starting platform
+    - The zoning tool will automatically detect vertices and snap to them ( blue indicators in the video below )
+    - Make sure you are always snapping to vertices when possible
+3. Select the opposite corner of the starting platform
+    - If the platform you're zoning is not rectangular see the [Freeform Mode](#freeform-mode) section
+4. Choose the height of your zone ( please follow the [zone height requirements](#zone-height))
+    - Make sure the **destination entity** is inside of your zone
+    - The entity will turn green when encompassed by the region
+    - The entity is usually above any starting platform ( grey box in the video below )
+5. If necessary, change the [Safe Height](#safe-height) of your zone
+
+{{<hint info>}}
+
+Zoning non-rectangular platforms: [Freeform Mode](#freeform-mode).  
+Zoning in mid-air: [Box Mode](#box-mode).
+
+{{</hint>}}
+
+{{<hint info>}}
+
+You can always [Edit Zones](#zone-editing) after creating them.
+
+{{</hint>}}
+
+{{<video src="/videos/map_zoning/create_start_zone.mp4">}}
+
+
+## Creating Stages and Checkpoints
+**Stages** and **Checkpoints** are used differently depending on the gamemode.  
+Please follow the guide appropriate for your map.  
+
+{{<hint info>}}
+
+Checkpoints should be placed where they can be hit consistently for better split comparisons.  
+Middle/End of a hallway is generally preferred  over placing them right at the start. 
+
+{{</hint>}}
+
+{{<hint info>}}
+
+When zoning **multiple routes** or **around a hole in geometry** you should be adding [Additional Regions](#multiple-regions) to a single zone.
```

Comment:
I would add to the end: "instead of creating extra zones". Especially since in other zoning systems people are used to, you create extra zones with the same number to do this.